ts/components/DevicePairingDialog.tsx

```diff
@@ -113,7 +118,7 @@ export class DevicePairingDialog extends React.Component<Props, State> {
                   </div>
                 ) : (
                   <div className="session-modal__centered">
-                    {renderPairedDevices}
+                    {'renderPairedDevices'}
```

Comment:
I guess you did that to remove the call to the function but it will print somewhere the string instead, right?

Reply:
That's right. Still in testing phase because I can't get a working phone to scan QR code.